WalletWasabi.Gui/Config.cs

```diff
@@ -27,6 +28,7 @@ public class Config : ConfigBase
 		public const int DefaultPrivacyLevelStrong = 50;
 		public const int DefaultMixUntilAnonymitySet = 50;
 		public const int DefaultTorSock5Port = 9050;
+		public const int DefaultJsonRpcServerPort = 18099;
```

Comment:
Please make a sister PR for this port. https://docs.wasabiwallet.io/building-wasabi/Ports.html